source/operators/translate/start_translate.py

```diff
@@ -104,7 +104,7 @@ def lambda_handler(event, context):
     for sentence in sentences:
         # Translate can handle 5000 unicode characters but we'll process no more than 4000
```

Comment:
Please update the comment to match the code.  Explain why its 1000 when the limit is 4000 so no one reverses the change on accident.

Reply:
I updated the comment, accordingly.